docs/sourcemap.asciidoc

```diff
@@ -66,4 +91,28 @@ request.post({url: 'http://localhost:8200/v1/client-side/sourcemaps',formData: f
     console.log('Sourcemaps uploaded!')
   }
 })
-----
\ No newline at end of file
+----
+
+You can also use `curl` command:
+
+[source,sh]
+----
+APPVERSION=`node -e "console.log(require('./package.json').version);"` && \
+curl http://localhost:8200/v1/client-side/sourcemaps -X POST \
+    -F sourcemap=@./dist/app.min.js.map \
+    -F service_version="$APPVERSION" \
+    -F bundle_filepath="http://localhost/app.min.js" \
+    -F service_name="service-name"
+----
+
+Or use current git commit hash:
+
+[source,sh]
+----
+APPVERSION=`git rev-parse --short HEAD` && \
+curl http://localhost:8200/v1/client-side/sourcemaps -X POST \
+    -F sourcemap=@./dist/app.min.js.map \
+    -F service_version="$APPVERSION" \
+    -F bundle_filepath="http://localhost/app.min.js" \
+    -F service_name="service-name"
```

Comment:
I am a bit confused about the usage of `app_version`. As we have renamed `app` to `service` wouldn't it be more consistent and easier to understand if you'd also use `service` over `app` locally?

Reply:
I think the root of confusion is because we are using service for what most people would call an "app" and I did voice my concern over this. So devs that come to this documentation, they are used to calling their frontend application an "app", but at some point we should make the transition and make the point that our configuration is called "service".

Reply:
What I'm saying is that the `service_version="$APPVERSION"` is there to show the user that they're suppose to use `service_version` for what they call "App version".

Reply:
I would have probably written it the following: 
```
SERVICE_VERSION=`git rev-parse --short HEAD` && \
curl http://localhost:8200/v1/client-side/sourcemaps -X POST \
   -F sourcemap=@./dist/app.min.js.map \
   -F service_version="$SERVICE_VERSION" \
   -F bundle_filepath="http://localhost/app.min.js" \
   -F service_name="service-name"
```
You are also using `service-name` and not `app-name`, so I don't see how this is consistent.
Anyways, you know the community better than I do, and if you think it is less confusing this way then it is fine with me.